docs/flinkDev/building.md

```diff
@@ -132,7 +132,7 @@ Since version 1.7 Flink builds with Scala version 2.11 (default) and 2.12.
 
 To build FLink against Scala 2.12, issue the following command:
 {% highlight bash %}
-mvn clean install -DskipTests -Dscala-2.12
+mvn clean install -DskipTests -Dscala-2.12 -Dscala.version=2.12.12
```

Comment:
Shouldn't we use the default scala version here?

Reply:
This is to show that you can use a custom version, which is the purpose of this PR. I think you only want to build it yourself to change the exact Scala binary version. Otherwise, you can just use the binary releases.

Reply:
Maybe put <scala_version>?

Reply:
I'll add a second example for building against a specific version and will use `<scala version>`